svc-task/tmessagebus/publish.go

```diff
@@ -36,12 +37,15 @@ func Publish(taskURI string, messageID string, eventType string) {
 
 	var eventID = uuid.NewV4().String()
 	var event = common.Event{
-		EventID:   eventID,
-		MessageID: messageID,
-		EventType: eventType,
+		EventID:        eventID,
+		MessageID:      messageID,
+		EventTimestamp: time.Now().Format(time.RFC3339),
+		EventType:      eventType,
+		Message:        "Task event",
```

Comment:
Update event message in accordance to what the event is about in task service

Reply:
updated